src/sandstorm/identity.capnp

```diff
@@ -23,13 +23,28 @@ using Util = import "util.capnp";
 using PermissionSet = List(Bool);
 # Set of permission IDs, represented as a bitfield.
 
+interface Identity {
+  # Represents a user identity.
+  #
+  # Things you can do:
+  # - Mention the identity on activity events to send notifications. See ActivityEvent in
+  #   activity.capnp.
+  # - Powerbox-request an Identity to have the user choose from among their contacts. Note that the
+  #   user will be prompted to share the grain with the target identity.
+  # - offer() the identity to the user in order to let them see the identity's profile card and
+  #   choose to add the indentity to their contacts. You could do this e.g. when the user clicks
+  #   on the identity's name in your app's UI. (TODO(someday): Not implemented yet.)
```

Comment:
Should there be a way to get a fresh `UserInfo` from an `Identity`? How else would an app be able to display up-to-date display names and avatars for users? (E.g. the collections app wants to display the avatar of the introducer user next to the grain.)